go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/prometheus/client_golang v1.9.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.18.0
-	github.com/prometheus/prometheus v1.8.2-0.20210321183757-31a518faab18
+	github.com/prometheus/prometheus v1.8.2-0.20210319192855-d614ae9ecf1c
```

Comment:
This is a downgrade. The changes required by this PR are already in, so no need to upgrade/downgrade Prometheus.